contrib/hashicorp/consul/benchmark_test.go

```diff
@@ -0,0 +1,75 @@
+package consul
+
+import (
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	consul "github.com/hashicorp/consul/api"
+)
+
+func BenchmarkKV(b *testing.B) {
```

Comment:
Since you did the great work of having these benchmarks, it would be nice to see the results. Would you mind posting them as a comment to this PR?

Lastly, I think your intention was to compare the performance with tracing and without. Please correct me if I'm wrong. In that case, maybe we should just benchmark the imported library and not our wrapper (even if counter productive). Then we would have two benchmarks: one of the original library with no tracing, and another of our wrapper, with tracing. That would give us the real comparison. For extra points, results could be posted using benchcmp: https://godoc.org/golang.org/x/tools/cmd/benchcmp

Reply:
Made suggested change, and added the benchmark results to the original PR